test/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+
+networks:
+  public:
+    name: scylla_rust_driver_public
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+        - subnet: 172.42.0.0/16
+services:
+  scylla1:
+    image: scylladb/scylla
+    ports:
+      - 127.0.0.1:9042:9042
+      - 127.0.0.1:9142:9142
+      - 127.0.0.1:19042:19042
+      - 127.0.0.1:19142:19142
+    volumes:
+      - ${GITHUB_WORKSPACE}:/workspace
+    networks:
+      public:
+        ipv4_address: 172.42.0.2
+    command: --rpc-address 172.42.0.2 --listen-address 172.42.0.2 --seeds 172.42.0.2 --skip-wait-for-gossip-to-settle 0 --thread-affinity false
```

Comment:
`--thread-affinity false` is redundant here, by default Scylla Docker instances use `--overprovisioned` which already disables thread affinity and does even more. See here: https://github.com/scylladb/scylladb/blob/master/dist/docker/commandlineparser.py